.gitignore

```diff
@@ -1,13 +1,14 @@
+# WebStorm
 .idea/
-.history/
```

Comment:
I would bring this one back. It's a known VSCode plugin that keeps local history changes.

Reply:
We could also use this template
https://www.toptal.com/developers/gitignore/api/webstorm,vscode,react

Reply:
Looks good to me!

Reply:
I'd make a new PR for that, however, so that we can merge the main topic of this PR already. Fine for you? If so, feel free to merge